go/Makefile

```diff
@@ -11,6 +11,15 @@ OASIS_GO ?= go
 #   migrate to a version of Go that has the fix, it can be removed.
 GOFLAGS ?= -trimpath -ldflags=-buildid= -v
 
+# Append git HEAD commit to the oasis node version, if git command exists and
+# this folder is in some git repository.
+GIT_REVISION = $(shell git rev-parse HEAD 2>/dev/null)
+ifneq ($(GIT_REVISION),)
+	# Truncate the commit hash and append +, if the branch was dirty.
+	VERSION_BUILD = $(shell git describe --always --match "" --dirty=+)
```

Comment:
Hm, why execute `git rev-parse HEAD` first and then `git describe ...`? Could you just do the describe or is there some reason why both are needed?

Reply:
I thought we would need full `$GIT_REVISION` also somewhere else. But I guess `$VERSION_BUILD` with potential dirty suffix + at the end is perfectly fine. Fixed.